proprietary/Components/src/denhaag/highlighted-links.tokens.json

```diff
@@ -0,0 +1,26 @@
+{
+  "denhaag": {
+    "highlighted-links": {
+      "background-color": { "value": "{denhaag.color.warmgrey.1}" },
+      "padding-block-start": { "value": "{denhaag.space.block.3xl}" },
+      "padding-block-end": { "value": "{denhaag.space.block.3xl}" },
+      "width": { "value": "100%" },
+      "container": {
+        "margin-inline-start": { "value": "auto" },
+        "margin-inline-end": { "value": "auto" },
+        "width": { "value": "100%" },
+        "max-width": { "value": "98rem" }
+      },
+      "hero-theme-page-highlighted-links": {
+        "padding-block-start": { "value": "4.375rem" }
+      },
+      "navigation-list": {
+        "display": { "value": "grid" },
+        "gap": { "value": "0 {denhaag.space.block.3xl}" },
+        "grid-template-columns": { "value": "repeat(auto-fill, minmax(22.75rem, 1fr))" },
```

Comment:
Correct me if i'm wrong. But if i check the design it's always 1, 2 or 3 columns. if thats the case, it's unnecessary to use a minmax()

Reply:
@paulAcato UX had specific column amount in mind per breakpoint.
There for scaling of column minimum widths is necessary, otherwise it won't behave with the specific amount of columns per breakpoint.

Reply:
`grid-template-columns: repeat(3, 1fr);` no minmax required for this ;-)

Reply:
I tested it, does'nt work without it.

Reply:
I tested it without the minmax, does not work with specific breakpoints.

Reply:
Check how i did it within the theme ;)